docker-compose.yml

```diff
@@ -0,0 +1,311 @@
+    expose:
+      - "5432"
+    restart: unless-stopped
+    stop_grace_period: 2m
+    stop_signal: SIGTERM
+    tty: true
+    volumes:
+      - ./compose-network/mirror-node/init.sh:/docker-entrypoint-initdb.d/init.sh
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-importer/application.yml
+      - ./compose-network/mirror-node/addressBook.bin:/usr/etc/hedera-mirror-importer/local-dev-1-node.addressbook.f102.json.bin
+
+  grpc:
+    image: gcr.io/mirrornode/hedera-mirror-grpc:0.49.0
+    container_name: mirror-node-grpc
+    restart: unless-stopped
+    mem_swappiness: 0
+    mem_limit: 512m
+    memswap_limit: 512m
+    depends_on:
+      network-node:
+        condition: service_started
+      db:
+        condition: service_started
+    environment:
+      HEDERA_MIRROR_GRPC_DB_HOST: db
+      SPRING_CONFIG_ADDITIONAL_LOCATION: file:/usr/etc/hedera-mirror-grpc/
+    networks:
+      - mirror-node-bridge
+    ports:
+      - "5600:5600"
+    volumes:
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-grpc/application.yml
+
+  importer:
+    image: gcr.io/mirrornode/hedera-mirror-importer:0.49.0
+    container_name: mirror-node-importer
+    restart: unless-stopped
+    mem_swappiness: 0
+    mem_limit: 768m
+    memswap_limit: 768m
+    depends_on:
+      network-node:
+        condition: service_started
+      db:
+        condition: service_started
+    environment:
+      HEDERA_MIRROR_IMPORTER_DB_HOST: db
+      SPRING_CONFIG_ADDITIONAL_LOCATION: file:/usr/etc/hedera-mirror-importer/
+    networks:
+      - mirror-node-bridge
+    volumes:
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-importer/application.yml
+      - ./compose-network/mirror-node/addressBook.bin:/usr/etc/hedera-mirror-importer/local-dev-1-node.addressbook.f102.json.bin
+
+  rest:
+    image: gcr.io/mirrornode/hedera-mirror-rest:0.49.0
+    container_name: mirror-node-rest
+    mem_swappiness: 0
+    mem_limit: 512m
+    memswap_limit: 512m
+    depends_on:
+      network-node:
+        condition: service_started
+      db:
+        condition: service_started
+    environment:
+      HEDERA_MIRROR_REST_DB_HOST: db
+    networks:
+      - mirror-node-bridge
+    ports:
+      - "5551:5551"
+    restart: unless-stopped
+    tty: true
+
+  web3:
+    image: gcr.io/mirrornode/hedera-mirror-web3:0.49.0
+    container_name: mirror-node-web3
+    mem_swappiness: 0
+    mem_limit: 512m
+    memswap_limit: 512m
+    depends_on:
+      network-node:
+        condition: service_started
+      db:
+        condition: service_started
+    networks:
+      - mirror-node-bridge
+    environment:
+      HEDERA_MIRROR_WEB3_DB_HOST: db
+      SPRING_CONFIG_ADDITIONAL_LOCATION: file:/usr/etc/hedera-mirror-web3/
+    ports:
+      - "8545:8545"
+    restart: unless-stopped
+    tty: true
+    volumes:
+      - ./compose-network/mirror-node/application.yml:/usr/etc/hedera-mirror-web3/application.yml
```

Comment:
This looks to be unneeded right now along with SPRING_CONFIG_ADDITIONAL_LOCATION.
```suggestion
```

Reply:
Got it ! Was added from mirror-node project. Removed.